src/Tribe/Importer/Featured_Image_Uploader.php

```diff
@@ -14,6 +14,16 @@ class Tribe__Events__Importer__Featured_Image_Uploader {
 	protected $featured_image;
 
 	/**
+	 * @var bool|array
+	 */
+	protected static $_attachment_guids_cache = false;
```

Comment:
Not a big problem, but I think we've generally been avoiding the `_prefix` convention, the access modifiers are enough by themselves.